modules/markup/html.go

```diff
@@ -146,7 +128,7 @@ type postProcessError struct {
 }
 
 func (p *postProcessError) Error() string {
-	return "PostProcess: " + p.context + ", " + p.Error()
+	return "PostProcess: " + p.context
```

Comment:
Shouldn't it be (maybe with nil check):
```suggestion
	return "PostProcess: " + p.context + ", " + p.err.Error()
```

Reply:
Sounds like a good idea.